app/views/pages/home.html.erb

```diff
@@ -28,30 +28,22 @@
           <% end %>
       </div>
       <div class="text-center space-y-8 md:grid md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 md:gap-8 xl:gap-8 md:space-y-0">
-        <div class="p-6 rounded-lg flex flex-col justify-center items-center">
-          <div>
-            <%= image_tag("pennsylvania-li.png", alt: "Pennsylvania L&I") %>
-          </div>
-          <h3 class="text-xl font-bold text-primary-700 mt-4">Pennsylvania L&I</h3>
-        </div>
-        <div class="p-6 rounded-lg flex flex-col justify-center items-center">
-          <div>
-            <%= image_tag("Alabama-OA.png", alt: "Alabama OA") %>
-          </div>
-          <h3 class="text-xl font-bold text-primary-700 mt-4">Alabama OA</h3>
-        </div>
-        <div class="p-6 rounded-lg flex flex-col justify-center items-center">
-          <div>
-            <%= image_tag("Massachusetts-DAS.png", alt: "Massachusetts DAS") %>
-          </div>
-          <h3 class="text-xl font-bold text-primary-700 mt-4">Massachusetts DAS</h3>
-        </div>
-        <div class="p-6 rounded-lg flex flex-col justify-center items-center">
-          <div>
-            <%= image_tag("Wisconsin-DWD.png", alt: "Wisconsin DWD") %>
-          </div>
-          <h3 class="text-xl font-bold text-primary-700 mt-4">Wisconsin DWD</h3>
-        </div>
+        <% @popular_states.each do |state| %>
+          <%= link_to occupation_standards_path(state: state.abbreviation) do %>
```

Comment:
@jenlu33 would you mind making a follow up PR that passes the parameters as: `q: "state:#{state.abbreviation}"`? We are accepting state as a parameter on its own, but if it is passed within `params[:q]`, then it will show up in the search box:

<img width="754" alt="Screen Shot 2023-08-23 at 12 59 46 PM" src="https://github.com/ApprenticeshipStandardsDotOrg/ApprenticeshipStandardsDotOrg/assets/1938665/afd19521-ba47-4b6c-9c03-3cf6c792ae2b">